### PR TITLE
[Feature] Allow admins to delete users from db

### PR DESCRIPTION
**Pull request by Freedisch**
## Description

This PR fixes issue #931 
This PR introduces a new delete button to users on user management page, allowing admin to delete users

## Demo

https://github.com/fedora-infra/anitya/assets/82499435/91ed7726-6dac-42a7-9bdc-7ec6d360e6b5

### CODE REVIEW

**Comment by Freedisch** [comment]
I'm taking care of the failing CI

**Comment by Zlopez** [comment]
I will wait till CI will be fixed, but the PR looks good :+1:

**Comment by Freedisch** [comment]
@Zlopez, I fixed the failing CIs, This PR is now ready for review

File: pyproject.toml [hunk 1]
```diff
@@ -119,6 +119,8 @@ Werkzeug = "3.0.1"
 toml = "^0.10.2"
 webargs = "^8.2.0"
 WTForms = "^3.0.1"
+attr = "^0.3.2"
```
Comment (Zlopez) [inline pyproject.toml:122]: Why adding those two dependencies? I don't see them imported anywhere.

**Review by Zlopez — CHANGES_REQUESTED** [review]
I see some imports I don't understand.

And also the conflicts needs to be fixed (maybe `master` branch in your fork is outdated).

File: pyproject.toml (hunk 1 above)
Comment (Freedisch) [reply pyproject.toml:122]: From the `Tox-docs CI`, There was an issue with `filters.py` inside `/workspace/src/github.com/fedora-infra/anitya/.tox/docs/lib/python3.11/site-packages/attr/.` This problem is occurring during an attempt to install idna, and attrs using Poetry. 
To fix that issue I installed manually both dependencies.

Screenshot 2024-03-13 at 14 20 44

Comment (Zlopez) [reply pyproject.toml:122]: This isn't related to Anitya project but it's issue in dependencies of pip or poetry. Those shouldn't be in the dependencies for Anitya as we don't use them. We should only update poetry.lock file.

Comment (Zlopez) [reply pyproject.toml:122]: Let me do that in separate PR.

Comment (Zlopez) [reply pyproject.toml:122]: The PR is now available https://github.com/fedora-infra/anitya/pull/1740. Could you remove changes in `poetry.lock` and `pyproject.toml` from this PR.

**Comment by Zlopez** [comment]
I also noticed that the news file is missing.

**Review by Zlopez — APPROVED** [review]

**Comment by Zlopez** [comment]
Thanks for the PR :-)

**Pull request merged**